python/cog/director/director.py

```diff
@@ -56,10 +57,12 @@ def __init__(
         self.predict_timeout = predict_timeout
         self.max_failure_count = max_failure_count
         self.report_setup_run_url = report_setup_run_url
+        self.idle = threading.Event()  # is_set() means a prediction is running!
```

Comment:
Is this comment right? From my reading of the code, it looks like a prediction is running when the event is cleared, and it's idle when set.

Reply:
ah yes I had it as `self.active` first but decided to invert